### DISCUSSION TITLE
Requesting data for creating org Admin on Sunbird Dev environment.

### DISCUSSION THREAD

**Discussion by omkar309** [Sunbird-Ed/Community#511] [Installation] [answered]
Hi team,
The **dev environment** that was available to Samagra was resettled when Angular update happened due to which all the old data might not exist. Hence we are **trying to create a new org admin ID and get a new channel ID for the same.**
Vaishali from EkStep has been helping us out and has shared a curl with us to create the Org admin for Samagra use cases to test. To create orgAdmin we need following header values,
--header 'Accept: application/json' \
--header 'Content-Type: application/json' \
--header 'X-Consumer-ID: X-Consumer-ID' \
--header 'ts: 2017-05-25 10:18:56:578+0530' \
--header 'X-msgid: 8e27cbf5-e299-43b0-bca7-8347f7e5abcf' \
--header 'X-Device-ID: X-Device-ID' \
--header 'x-authenticated-user-token: xxx' \
--header 'Authorization: Bearer xxx' \

Requesting anyone who can provide us with the data to come forward and help us with the same or point us to the right individual who could help. This will help us access the dev environment and continue with our development.

CC: @gandham-santhosh @abhay-samagra

**Comment by chitranshu-keshav** [comment]
@vinukumar-vs @vaivk369 can you please guide @omkar309 on how to get the required details.

**Comment by vaivk369** [comment]
@vinukumar-vs  Can you please help here?

**Comment by vaivk369** [comment, marked as answer]
@omkar309 Have shared the auth bearer token and the CURL to generate the x-authenticated-user-token with Rekha. Please check.

**Reply by omkar309** [reply]
Thanks @vaivk369. Will check and post an update.

**Reply by chitranshu-keshav** [reply]
@omkar309 can you please mark this thread as answered

**Reply by omkar309** [reply]
Hey @chitranshu-keshav our issue has not been resolved fully yet, active work is going on.
Progress  summary: 
- Users are created and joined the organisation.
- But still content type available is "Practice Set". And as per @vaivk369 's suggestion, primary category has been created but the issue still persists.
- So, have contacted @vaivk369 for further procedure or to share any curl that can help us update our category into channel read API. She's helping with the same.
